doc/proxy-clients.rst

```diff
@@ -28,6 +28,8 @@ You also need a `PSR-7 message implementation`_. If you use Guzzle 6, Guzzle’s
 implementation is already included. If you use another client, install one of
 the implementations. Recommended:
 
+TODO: i think we need the matching adapter for the factory, if guzzle6-adapter is not installed
```

Comment:
@ddeboer you did the first iteration of this and wrote this doc. are you sure this will work without the php-http adapter for guzzle?